crypto/bn/bn_intern.c

```diff
@@ -68,9 +68,9 @@ signed char *bn_compute_wNAF(const BIGNUM *scalar, int w, size_t *ret_len)
     }
     window_val = scalar->d[0] & mask;
     j = 0;
-    while ((window_val != 0) || (j + w + 1 < len)) { /* if j+w+1 >= len,
-                                                      * window_val will not
-                                                      * increase */
+    while (window_val != 0 || j + w + 1 < len) { /* if j+w+1 >= len,
```

Comment:
Another subjective example. To me personally `(j + w + 1) < len` would be more readable. Well, if not for styling requirement to have spaces everywhere, I otherwise would argue in favour of `j+w+1 < len` :-)

Reply:
I'd pefer `(j + w + 1) < len` as well.